pkgs/development/tools/misc/texinfo/texinfo-disable-manpages.patch

```diff
@@ -0,0 +1,60 @@
+--- texinfo-6.5/man/Makefile.am	2015-11-10 00:50:56.000000000 +0200
++++ texinfo-6.5/man/Makefile.am	2018-11-30 09:37:33.619678388 +0300
+@@ -22,13 +22,13 @@
+ endif
+ 
+ # These are hand-written.
+-man_MANS += info.5 texinfo.5
++#man_MANS += info.5 texinfo.5
```

Comment:
Can you explain why we cannot cross-compile this?

Reply:
Help2man runs the program to generate the man page. See coreutils for prior art of pulling the man pages from the native version.

Reply:
Ive fixed some of them (.1's) but cannot fix those (.5's). I got "no rules to make ginfo.5" err.